backend/src/modules/place/mutation.js

```diff
@@ -205,7 +136,7 @@ export const deleteTag = async (_, { name }, { dbConnection, auth }) => {
   }
 };
 
-//-------SOLUTION FOR LAST SPRINT------
+//TODO:-------SOLUTION FOR LAST SPRINT------
 // export const addSportType = async (_, stid, {dbConnection, auth}) => {
```

Comment:
Všechny komentáře bych smazal kvůli čistotě kodu. Určitě za to budou i strhávat body.